generator/uscore/metadata_extractor.rb

```diff
@@ -320,6 +336,10 @@ def set_first_search(sequence, params)
         sequence[:searches].delete(search)
         sequence[:searches].unshift(search)
       end
+
+      def capitalize_first_letter(str)
```

Comment:
Why is this needed over [`#capitalize`](https://ruby-doc.org/core-2.5.4/String.html#method-i-capitalize)?